### PR TITLE
Start refactoring <select>

### PR DESCRIPTION
**Pull request by AlecRust**
### Current problem

Continuing from #2902 there are improvements to select inputs it would be nice to fix before tackling the table-based layouts of these forms:
- They don't have a real associated label
- The fake label is in a different table row
- They use a fixed width not suited to mobile-first forms
- The position of the background image is broken in some cases

### Proposed solution

- Add new improved ".form-select" CSS
- Replace fake labels with real label elements
- Move label in to same table row as select

Since this is a fairly large change to review in itself, I haven't done the full migration in this PR. This PR updates:
- Add/edit user forms
- Add/edit cron forms
- Add/edit web forms
- Edit server form

This is perhaps 50% of all select fields. The old ".vst-list" styles remain in place until the migration is complete.

### Screenshots

Very minor visual changes, you'd need to know what you were looking for to spot them.

**Before:**
image

**After:**
image

### CODE REVIEW

File: web/templates/pages/edit_web.html [hunk 1]
```diff
@@ -261,15 +250,12 @@
 											</div>
 										</td>
 									</tr>
+									<!-- TODO: Why is this here/how does it get shown? -->
```
Comment (AlecRust) [inline web/templates/pages/edit_web.html]: This "Home Directory" `<select>` is strange, it seems to be always hidden. Is that intentional, or a bug?

Comment (jaapmarcus) [reply web/templates/pages/edit_web.html]: Old vesta code *** Just remove it

Comment (AlecRust) [reply web/templates/pages/edit_web.html]: OK, I've removed this in 5fee68b96.

**Review by jaapmarcus — COMMENTED** [review]
Changes look fine we should consider "updating" add/web/index.php and remove the non visible options

**Review by jaapmarcus — APPROVED** [review]

File: web/templates/pages/edit_web.html [hunk 2]
```diff
@@ -250,17 +250,6 @@
 											</div>
 										</td>
 									</tr>
-									<!-- TODO: Why is this here/how does it get shown? -->
```
Comment (jaapmarcus) [inline web/templates/pages/edit_web.html]: https://github.com/hestiacp/hestiacp/blob/5a0a002197eab45157784eaa4b8900bd5d32923f/web/edit/web/index.php#L410-L421

Should also be removed then

Comment (jaapmarcus) [reply web/templates/pages/edit_web.html]: This one is resolved.

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**